src/utils/schema.ts

```diff
@@ -28,3 +32,47 @@ export function parseSchemaResult(schema: z.ZodTypeAny, v: unknown) {
   }
   return schemaResult;
 }
+
+export async function validateAnt({
```

Comment:
i go back and forth on if these can/should be static functions on the class, or as utilities. we're following the pattern with spawn, so fine to leave - but let's add a TODO on if we stack the ANT class with these utility functions as opposed to exporting them as standalone. what's your opinion?

Reply:
I would actually like spawn to be a static member on the ANT class - similarly on the other classes we have for the ant registry and IO process. We have wanted a `clone` as well which is somewhat related.

Considering your other comment on passing in the ant instance - and I do agree thats a better way - then it would actually make sense to make this a non-static method on the AntReadable class though.